File: TiendaVirtual.Core/Entities/MensajesModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TiendaVirtual.Core.Entities
{
    public class MensajesModel
    {
        public int UsuarioIDOrigen { get; set; }

        public int UsuarioIDDestino { get; set; }

        public string Mensaje { get; set; }
    }
}

File: TiendaVirtual.Client/ClientApp/src/app/Modelos/MensajesModel/MensajesModel.ts
export class MensajesModel{

  constructor(MensajeID: number, Mensaje: string, UsuarioIDDestino: number, UsuarioIDOrigen: number, PedidoID: number, EstadoMensaje: boolean) {
    this.MensajeId = MensajeID,
      this.Mensaje1 = Mensaje,
      this.UsuarioIdDestino = UsuarioIDDestino,
      this.UsuarioIdOrigen = UsuarioIDOrigen,
      this.PedidoId = PedidoID,
      this.EstadoMensaje = EstadoMensaje
  }

  MensajeId: number;
  Mensaje1: string;
  UsuarioIdDestino: number;
  UsuarioIdOrigen: number;
  PedidoId: number;
  EstadoMensaje: boolean;
}
